sqlitedict.py

```diff
@@ -303,7 +315,13 @@ def terminate(self):
 
     def __del__(self):
         # like close(), but assume globals are gone by now (do not log!)
-        self.close(do_log=False)
+        try:
+            self.close(do_log=False)
+        except Exception:
+            # prevent error log flood in case of multiple SqliteDicts
+            # closed after connection lost (exceptions are always ignored
+            # in __del__ method.
+            pass
```

Comment:
Can we print the exception to sys.stderr?
As in [ del docs](https://docs.python.org/2/reference/datamodel.html#object.__del__)
> exceptions that occur during their execution are ignored, and a warning is printed to sys.stderr instead.